README.md

```diff
@@ -39,3 +39,52 @@ This will create a container called `fedora-toolbox-<version-id>`.
 [user@hostname ~]$ toolbox enter
 ⬢[user@toolbox ~]$
 ```
+
+## Distro support
+
+By default, Toolbox creates the container using an
+[OCI](https://www.opencontainers.org/) image called
+`<ID>-toolbox:<VERSION-ID>`, where `<ID>` and `<VERSION-ID>` are taken from the
+host's `/usr/lib/os-release`. For example, the default image on a Fedora 30
+host would be `fedora-toolbox:30`.
+
+This default can be overridden by the `--image` option in `toolbox create`,
+but operating system distributors should provide an adequately configured
+default image to ensure a smooth user experience.
+
+## Image requirements
+
+Toolbox customizes newly created containers in a certain way. This requires
+the following tools and directories to be present inside the OCI image:
+* `/etc/krb5.conf.d`: optional, for Kerberos
+* `getent(1)`
+* `id(1)`
+* `ln(1)`
+* `mkdir(1)`: for hosts where `/home` is a symbolic link to `/var/home`
+* `passwd(1)`
+* `readlink(1)`
+* `rm(1)`
+* `rmdir(1)`: for hosts where `/home` is a symbolic link to `/var/home`
+* `sleep(1)`
+* `test(1)`
+* `touch(1)`
+* `unlink(1)`
+* `useradd(8)`
+
+The image should have `sudo(8)` enabled for users belonging to either the
+`sudo` or `wheel` groups, and the group itself should exist. File an
+[issue](https://github.com/containers/toolbox/issues/new) if you really need
+support for a different group. However, it's preferable to keep this list as
+short as possible.
+
+Since Toolbox only works with OCI images that fulfill certain requirements,
+it will refuse images that aren't tagged with
+`com.github.containers.toolbox="true"` and
```

Comment:
Yes, you should have both until Fedora 29 goes end-of-life. Tentatively that's 29th November.

It's caused by the project changing its identity from `debarshiray/fedora-toolbox` to `debarshiray/toolbox` to `containers/toolbox`. See commit de5e5df9b793edc746cde7125cd437b69f15ccd7 for further details.

Maybe we can mention this? Or just not mention the old label since 29th November isn't that far away?

Reply:
Maybe we should mandate the newer, suggest the older for full compatibility and check for either in toolbox (both do not need to be present) to allow migrating to the newer one.

Reply:
Yes, that's a good idea.

Sadly, due to the limitations of POSIX shell as a programming environment, supporting an extra label surprisingly explodes the code by quite a bit in certain cases. For historical reasons it also supports `com.redhat.component` alongwith `com.github.debarshiray.toolbox`, so `com.github.containers.toolbox` will be the third.

It's not super difficult to fully support a third label, but we are also at a point that I'd really want to increase the priority of our rewrite away from POSIX shell over all non-critical problems. That's why I am hesitant to spend time hacking up the code and testing all the fallback paths.

(By critical problems, I am roughly thinking of better recovery and error handling from catastrophic Podman breakage, making it easier to debug failures, making room for `sudo toolbox ...`, etc.)

Realistically speaking, if we are lucky, we can drop `com.redhat.component` when F29 goes EOL in roughly a month. So at that point, adding a new label is just a matter of search-and-replace instead of writing new code.

So, it's one extra line that image maintainers need to have for the next one month, versus spending a bunch of time adding and testing fallback paths that would be gone after a month. :)